### PR TITLE
YT-15297 Reduce font for envelope id

### PR DESCRIPTION
**Pull request by pramodtech**
Waiting PM approval

### CODE REVIEW

File: lib/docusign_rest/client.rb [hunk 1]
```diff
@@ -1674,7 +1674,7 @@ def get_envelope_id_options(input)
 
       {
         add_envelope_tab: true,
-        envelope_id_tabs: [{ anchorString: input[:anchor_string], bold: false }],
+        envelope_id_tabs: [{ anchorString: input[:anchor_string], bold: false, fontSize: 'size8' }],
```
Comment (chiragsinghal) [inline lib/docusign_rest/client.rb:1677]: is the size case sensitive? In documentation it has `Size8` instead of `size8`

**Review by chiragsinghal — APPROVED** [review]
A minor question, rest looks good

File: lib/docusign_rest/client.rb (hunk 1 above)
Comment (pramodtech) [reply lib/docusign_rest/client.rb:1677]: Not case sensitive, verified it works.

**Comment by pramodtech** [comment]
@chiragsinghal merging this now. It is minor change and if PM do not approve it can be reverted easily.

**Pull request merged**